### PR TITLE
Fixed Grammer

### PR DESCRIPTION
**Pull request by RajdeepPy**
## Fixes #.
Fixed some grammatical errors on README.md, ApplicationArchitecture.md, ManuelTests.md, NewFeatureProcess.md, Roadmap.md

### Description of the changes:
- Pretty self-explanatory -- you could view the grammatical changes that make the sentence flow better for each change.

### How changes were validated:

- I proofread all the changes I made 4 times.

### CODE REVIEW

**Comment by jsoref** [comment]
fwiw, it's `grammar` ...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,11 +3,11 @@ The Calculator team encourages community feedback and contributions. Thank you f
 making Calculator better! There are several ways you can get involved.
 
 ## Reporting issues and suggesting new features
-If Calculator is not working properly, please file a report in the [Feedback Hub](https://insider.windows.com/en-us/fb/?contextid=130&newFeedback=True).
+If the Calculator is not working properly, please file a report in the [Feedback Hub](https://insider.windows.com/en-us/fb/?contextid=130&newFeedback=True).
```
Comment (jsoref) [inline CONTRIBUTING.md]: Calculator is a proper noun, this kind of change is incorrect.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -45,12 +45,12 @@ components, prefer the patterns described in the [C++ core guidelines](http://is
 and the [modern C++/WinRT language projections](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/).
 
 ### Testing
-Your change should include tests to verify new functionality wherever possible. Code should be
+Your change should include tests to verify new functionality wherever possible. The code should be
```
Comment (jsoref) [inline CONTRIBUTING.md]: I'm -0 on this. "Code" can be used w/o an article.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ assign the request to an appropriate reviewer within two days. Any member of the
 participate in the review, but at least one member of the Calculator team will ultimately approve
 the request.
 
-Often, multiple iterations will be needed to responding to feedback from reviewers. Try looking at
+Often, multiple iterations will be needed to respond to feedback from reviewers. Try looking at
```
Comment (jsoref) [inline CONTRIBUTING.md]: 👍

File: README.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ Calculator ships regularly with new features and bug fixes. You can get the late
 
 ## Features
 - Standard Calculator functionality which offers basic operations and evaluates commands immediately as they are entered.
-- Scientific Calculator functionality which offers expanded operations and evaluates commands using order of operations.
+- Scientific Calculator functionality which offers expanded operations and evaluates commands using the order of operations.
```
Comment (jsoref) [inline README.md]: 👍

**Comment by ZekeMedley** [comment]
I believe that "Calculator" in this case is a proper noun. They are referring to the application. This makes statements like "Calculator is  ..." a legal move the same way "Microsoft is ..." is legal.

File: docs/ApplicationArchitecture.md [hunk 5]
```diff
@@ -56,22 +56,22 @@ Once the window is resized to have enough space, the panel becomes docked along
 
 #### Scientific mode, inverse function button presence
 
-In the Scientific mode, for small window sizes there is not enough room to show all the function buttons. The mode
+In the Scientific mode, for small window sizes, there is not enough room to show all the function buttons. The mode
```
Comment (jsoref) [inline docs/ApplicationArchitecture.md]: 👎 See [Comma Placement Level 2: Restrictive vs. Nonrestrictive Clauses](https://www.grammarly.com/blog/subordinate-clause/)

File: docs/ApplicationArchitecture.md [hunk 6]
```diff
@@ -56,22 +56,22 @@ Once the window is resized to have enough space, the panel becomes docked along
...
 hides some of the buttons and provides a Shift (↑) button to toggle the visibility of the collapsed rows. When the
 window size is large enough, the buttons are re-arranged to display all function buttons at the same time.
 
 <img src="Images\VisualStates\Scientific1.gif" height="400" />
 
 #### Unit Converter aspect ratio adjustment
 
-In the Unit Converter mode, the converter inputs and the numberpad will re-arrange depending on if the window is in
+In the Unit Converter mode, the converter inputs and the number pad will re-arrange depending on if the window is in
```
Comment (jsoref) [inline docs/ApplicationArchitecture.md]: -0. I didn't change this in #135.

File: docs/ManualTests.md [hunk 7]
```diff
@@ -124,13 +124,13 @@ Steps:
     m.	“( )" Parenthesis
 
 
-**All Calulators Test: Verify memory functions**
+**All Calculators Test: Verify memory functions**
```
Comment (jsoref) [inline docs/ManualTests.md]: -0. I fixed this in #135, and I'd rather that be merged. (I'm a random contributor like you, but this PR is much more complicated and will require iteration.)

File: docs/ManualTests.md [hunk 8]
```diff
@@ -124,13 +124,13 @@ Steps:
...
 Steps:
 1.	Launch the "Calculator" app.
 2.	Navigate to "Standard" Calculator.
 3.	Perform a calculation and press the MS button.
-4.	If small scale, Select the (M) with the drop down arrow
-*Expected: Calculation from previous step is present.*
+4.	If small scale, Select the (M) with the drop-down arrow
+*Expected: Calculation from the previous step is present.*
```
Comment (jsoref) [inline docs/ManualTests.md]: 👍

File: docs/ManualTests.md [hunk 9]
```diff
@@ -210,7 +210,7 @@ Steps:
 *Expected: All buttons are present and the up arrow is grayed out.*
 6.	For Scientific Mode: At a Smaller Scale
 *Expected: All buttons are present and the up arrow is able to be toggled.*
-7.	For Programmer Mode: At a Any Scale
+7.	For Programmer Mode: At an Any Scale
```
Comment (jsoref) [inline docs/ManualTests.md]: 👎 probably s.b. "At Any Scale"

File: docs/ManualTests.md [hunk 10]
```diff
@@ -237,11 +237,11 @@ Verify the following:
 For Programmer Mode
 Verify the following:
 6.	"Bit Toggling Keypad":
-*Expected: In app keypad changes to represent Bits (1s and 0s).*
+*Expected: In-app keypad changes to represent Bits (1s and 0s).*
```
Comment (jsoref) [inline docs/ManualTests.md]: 👍 (this matches 5 other places in this file)

File: docs/ManualTests.md [hunk 11]
```diff
@@ -237,11 +237,11 @@ Verify the following:
...
 7.	"QWORD / DWORD / WORD / BYTE":
 *Expected: Toggles as expected.*
 8.	"Hex" Hexadecimal:
-*Expected: A B C D E F become active and user can use them. A maximum of 16 characters can be entered.*
+*Expected: A B C D E F become active and the user can use them. A maximum of 16 characters can be entered.*
```
Comment (jsoref) [inline docs/ManualTests.md]: -0, [the] is implied and it's ok in semi-informal English to omit

File: docs/NewFeatureProcess.md [hunk 12]
```diff
@@ -27,7 +27,7 @@ It will typically include these sections:
   customer need or pain point we need to remedy? Is there a business goal or metric we are trying
   to improve? Do we have a hypothesis we want to prove or disprove?
 * **Evidence or User Insights**: Why should we do this? Potential sources of data: Feedback Hub,
-  other GitHub issues, other anecdotes from listening to customers in person or online, request
+  other GitHub issues, other anecdotes from listening to customers in person or online, a request
```
Comment (jsoref) [inline docs/NewFeatureProcess.md]: 👎

File: docs/NewFeatureProcess.md [hunk 13]
```diff
@@ -72,7 +72,7 @@ addressed:
 - [ ] Does the design include provisions for [all users](https://docs.microsoft.com/en-us/windows/uwp/design/accessibility/designing-inclusive-software)
   and [all cultures](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/guidelines-and-checklist-for-globalizing-your-app)?
 - [ ] Is it technically feasible to build this feature? Take a look at the "before committing"
-  checklist below and identify any issues which are likely to be blockers.
+  the checklist below and identify any issues which are likely to be blockers.
```
Comment (jsoref) [inline docs/NewFeatureProcess.md]: 👎  `the "before committing" checklist` already has an article.

File: docs/ManualTests.md (hunk 11 above)
Comment (rbeesley) [reply docs/ManualTests.md]: "and user can use" vs. "and the user can use"... I think that adding "the" makes it read better, especially with "user" and "use" being so close together in the same sentence.

**Comment by mcooley** [comment]
Hi @RajdeepPy, thanks for your interest in this repo. For PRs which primarily make small improvements to comments or documentation, we'll typically consider them only if they fix objective defects in the existing documentation and only if they're easy to merge. Some of the changes here might be okay, while others should be left out (as others have mentioned).

I'm going to close this PR for now, but if you'd like to submit a more limited set of changes we'd take another look.

**Pull request closed without merging**